cmd/installer/cli/install_test.go

```diff
@@ -560,6 +561,29 @@ func Test_runInstallAPI(t *testing.T) {
 	certPool := x509.NewCertPool()
 	certPool.AddCert(cert.Leaf)
 
+	// We need a release object to pass over to the Web component.
+	dataMap := map[string][]byte{
+		"kots-app.yaml": []byte(`
```

Comment:
should probably include an icon and a title

Reply:
we can, albeit it's not really being used right now. Unless we also want to run some requests within this test against the web components?